src/coreclr/jit/instr.h

```diff
@@ -47,6 +51,14 @@ enum instruction : unsigned
 
     INS_lea,   // Not a real instruction. It is used for load the address of stack locals
 
+#elif defined(TARGET_LOONGARCH64)
+    #define INSTS(id, nm, fp, ldst, fmt, e1) INS_##id,
+    #include "instrs.h"
+
+    //INS_dneg,  // Not a real instruction. It will be translated to dsubu.
+    //INS_neg,   // Not a real instruction. It will be translated to subu.
+    //INS_not,   // Not a real instruction. It will be translated to nor.
```

Comment:
Remove these comments. We should include even "pseudo-instructions", because the encoder can do the correct encoding.